torch/_dynamo/repro/after_dynamo.py

```diff
@@ -451,13 +451,10 @@ def repro_run(options, mod, load_args):
     else:
         with torch.amp.autocast("cuda", enabled=options.autocast):
             args = run_load_args(options, mod, load_args)
-            ref = run_fwd_maybe_bwd(
-                mod, args, only_fwd=options.only_fwd, disable_clone=True
-            )
-            del args
```

Comment:
Are you sure we can safely remove del args? That might have some fun memory bubble issues if `run_fwd_maybe_bwd` is expensive.